src/hooks/useGitHub/useGitHub.js

```diff
@@ -21,32 +21,24 @@ const useGitHub = () => {
     const profileName = state.profileToSearch
     axios.get(`${baseURL}${profileName}`)
       .then(res => {
-        dispatch({
-          type: 'RESULT_IS_PROFILE',
-          payload: res.data,
-        })
-        return axios.get(`${baseURL}${profileName}/repos`)
-      })
-      .then(res => {
-        const curatedRepos = res.data.map(repo => {
-          return {
-            name: repo.name,
-            description: repo.description,
-            liveDemo: repo.homepage,
-            url: repo.html_url,
-            updatedAt: repo.updated_at,
-            languagesUrl: repo.languages_url,
-          }
-        })
-
-        dispatch({
-          type: 'RESULT_OF_REPOS',
-          payload: curatedRepos,
-        })
+        const response = res.data
+        console.log(res.data)
+        if(response.profile == null) {
```

Comment:
Is it really necessary to use `==`? Also is it necessary to compare with `null`? Is `if(response.profile)` enough
```suggestion
        if(response.profile === null) {
```